assets/src/helpers.js

```diff
@@ -396,14 +396,32 @@ export const filterBlockAttributes = ( blockAttributes, blockType, innerHTML ) =
  *
  * @param {Object} element
  * @param {Object} blockType
+ * @param {Object} attributes
  *
  * @return {Object} The element.
  */
-export const wrapBlocksInGridLayer = ( element, blockType ) => {
+export const wrapBlocksInGridLayer = ( element, blockType, attributes ) => {
 	if ( ! element || ! ALLOWED_MOVABLE_BLOCKS.includes( blockType.name ) ) {
 		return element;
 	}
 
+	// If both rotation and animation are present then we'll need to use a separate wrapper for rotation.
```

Comment:
Can't we just always use a separate wrapper? Would make maintenance a bit easier I think.

Reply:
Yes, we could, I was trying to avoid adding additional wrappers to the core blocks if necessary since animation + rotation isn't necessarily a super common use case. However, if you think it's better to always add a wrapper, I can change that.